routing/pathfind_test.go

```diff
@@ -2224,6 +2224,52 @@ func TestNoCycle(t *testing.T) {
 	}
 }
 
+// TestRouteToSelf tests that it is possible to find a route to the self node.
+func TestRouteToSelf(t *testing.T) {
+	t.Parallel()
+
+	testChannels := []*testChannel{
+		symmetricTestChannel("source", "target", 100000, &testChannelPolicy{
+			Expiry: 144,
+		}, 1),
```

Comment:
Add some more test channels to find a useful route for rebalancing purposes?

Reply:
Added a triangular test route